### PR TITLE
GEN-2069 -styles(mobile-comparison-table): update design

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Updated Mobile Comparison Table UI/UX

Screenshot 2024-05-16 at 17 42 07

## Justify why they are needed

Part of Widget + Car project

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4283?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4283** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4283?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4282** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4282?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4281** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4281?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/ComparisonTable/ComparisonTable.tsx [hunk 1]
```diff
@@ -40,12 +53,37 @@ export function Header({ children, active, ...props }: HeaderProps) {
...
+  )
+  const Children = description ? (
+    <Collapsible.Root>
+      <Collapsible.Trigger className={collapisableTrigger}>
+        <ChevronIcon className={triggerIcon} />
+        {Title}
+      </Collapsible.Trigger>
+
+      <Collapsible.Content className={collapisableContent}>
+        <Text className="description" as="span" size="xs" color="textSecondary">
```
Comment (alebedev) [inline apps/store/src/components/ComparisonTable/ComparisonTable.tsx:75]: Are we relying on "description" CSS class somehow?  Looks odd

File: apps/store/src/components/ComparisonTable/ComparisonTable.css.ts [hunk 2]
```diff
@@ -74,3 +69,26 @@ export const aligner = styleVariants({
     justifyContent: 'flex-end',
   },
 })
+
+export const collapisableTrigger = style({
```
Comment (alebedev) [inline apps/store/src/components/ComparisonTable/ComparisonTable.css.ts:73]: `collapisable` -> `collapsible`.  Here and few other places

**Review by alebedev — APPROVED** [review]

File: apps/store/src/components/ComparisonTable/ComparisonTable.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/components/ComparisonTable/ComparisonTable.tsx:75]: Nope, that thing was forgotten there. Thanks

**Comment by guilhermespopolin** [comment]
### Merge activity

* **May 17, 5:44 AM EDT**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4283).

**Pull request merged**